### ISSUE TITLE
Poor performance of instant-ngp

### ISSUE THREAD

**Issue by junha-l** [nerfstudio-project/nerfstudio#1560]
**Describe the bug**
Hi, I encounter a similar problem with #1384 while training instant-ngp on mipnerf360 v2 dataset. 

The same command produces different results with nerfstudio v0.1.18 and v0.1.6 (as suggested by @ksnzh).
```ns-train instant-ngp --vis wandb --data ../nerfstudio/data/nerf_360_v2/bicycle nerfstudio-data --downscale-factor 8```

![fig](https://user-images.githubusercontent.com/39543958/223186956-7ef0ea13-670f-44b1-8595-7c22f2bd2883.png)
As can be seen in the graph, the training procedure is somehow unstable with nerfstudio v0.1.18. 

I tried to find any difference in configs between the two versions but failed. 
Could anyone help me with this? Thanks in advance!

**To Reproduce**
1. preprocess mipnerf360 dataset
```ns-process-data images --data data/nerf_360_v2/bicycle/images --skip_colmap --skip_image_processing --colmap_model_path sparse/0 --verbose  --output-dir data/nerf_360_v2/bicycle```
2. train instant-ngp at nerfstudio v0.1.18
```ns-train instant-ngp --vis wandb --data ../nerfstudio/data/nerf_360_v2/bicycle nerfstudio-data --downscale-factor 8```
3. train instant-ngp again w/ nerfstudio v0.1.6
```ns-train instant-ngp --vis wandb --data ../nerfstudio/data/nerf_360_v2/bicycle nerfstudio-data --downscale-factor 8```

**Expected behavior**
Produce similar results.

**Screenshots**
- rendered eval image with nerfstudio v0.1.6 at iter 10k
![10k_v0 1 6](https://user-images.githubusercontent.com/39543958/223188964-1c6954f4-3236-4f30-9d17-3c71f8a2fb17.png)
- rendered eval image with nerfstudio v0.1.18 at iter 10k
![10k_v0 1 18](https://user-images.githubusercontent.com/39543958/223189038-a9327139-b08e-41e7-984e-55fc934a1c9a.png)
- rendered eval image with nerfstudio v0.1.6 at iter 29k
![29k_v0 1 6](https://user-images.githubusercontent.com/39543958/223189125-fd8582f4-6256-49be-921c-b5aefec00ef9.png)
- rendered eval image with nerfstudio v0.1.18 at iter 29k
![29k_v0 1 18](https://user-images.githubusercontent.com/39543958/223189252-f2e66fe9-4fa9-4c43-a1bb-46e9fe092e44.png)

**Comment by junha-l** [comment]
I found that downgrading ```nerfacc``` from v0.3.5 to v0.2.1 resolves the problem. 
Seems that there are some breaking changes in ```nerfacc```, not in ```nerfstudio```.

**Comment by junha-l** [comment]
Hi @zhan-xu, I only downgraded nerfacc and ended up changing a few lines of code to get things working again. The thing is that the output of ```nerfacc.ray_marching``` function has been changed. I recommend you to check out this part.

**Comment by zhan-xu** [comment]
thank you @junha-l . Will do.

**Comment by 97littleleaf11** [comment]
cc @ruilong

**Comment by liruilong940607** [comment]
oh thanks for looping me in @97littleleaf11! I will look into this shortly

**Comment by liruilong940607** [comment]
@97littleleaf11 Yeah we are working on a major upgrade to nerfacc. 0.4.0 is the first part of this major update and more are coming. But I'm still curious what causes this degradation

**Comment by machenmusik** [comment]
FYI, I just tried with
`"nerfacc @ git+https://github.com/KAIR-BAIR/nerfacc.git@17e28de",`
on a dataset I can't share which looks fine with nerfacto,
and it looks pretty bad with instant-ngp.

**Comment by liruilong940607** [comment]
I think the current usage of instant-ngp in this codebase is still based on single level occupancy grid with contraction. While the lastest update in nerfacc switches to multi-level occupancy grid and achieves much better results. 

In any cases, it should not dramatically fail with proper set of hyper-parameters, which definitely need to set on per scene basis, especially the render step size: 
https://github.com/nerfstudio-project/nerfstudio/blob/2b8140b583df96a52218f71170191e8de7ca4293/nerfstudio/models/instant_ngp.py#L74-L81
